packages/harmony/src/components/button/BaseButton.tsx

```diff
@@ -16,37 +16,25 @@ import type { BaseButtonProps } from './types'
 export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
   function BaseButton(props, ref) {
     const {
-      text,
       iconLeft: LeftIconComponent,
       iconRight: RightIconComponent,
       disabled,
       isLoading,
       widthToHideText,
```

Comment:
General question: I'm not quite understanding this prop, when would we want to hide text? (I probably need to tweak the aria label logic that I removed below)